iron-overlay-manager.html

```diff
@@ -36,7 +39,42 @@
         return this._backdropElement;
       }.bind(this)
     });
-  }
+
+    /**
+     * The deepest active element.
+     * returns {?Node} element the active element
+     */
+    this.deepActiveElement = null;
+    Object.defineProperty(this, 'deepActiveElement', {
```

Comment:
I made it a getter to "seem" more similar to `document.activeElement`, other than that no particular reason.